src/types/num.rs

```diff
@@ -420,6 +422,49 @@ mod bigint_conversion {
             let zero: BigInt = FromPyObject::extract(fib.call1("fib", (0,)).unwrap()).unwrap();
             assert_eq!(zero, BigInt::from(0));
         }
+
+        /// `OverflowError` on converting python int to BigInt
+        #[test]
+        fn issue_629() {
+            let gil = Python::acquire_gil();
+            let py = gil.python();
+            fn test<T>(value: T, py: Python)
```

Comment:
How about change this function into macro?
No one wants to read such a long trait bound in test functions.

Reply:
good idea